stack.yaml

```diff
@@ -1,10 +1,19 @@
-resolver: lts-17.7
+resolver: lts-18.8
 packages:
-- .
-- example-project
+  - .
+  - example-project
 nix:
   packages:
     - zlib.dev
     - zlib.out
 # Work around https://github.com/commercialhaskell/stack/issues/5290
 save-hackage-creds: false
+extra-deps:
+  - github: brendanhay/amazonka
+    commit: 098795f847df7239af64535c6b2b233f3fbd699c
+    subdirs:
+      - lib/amazonka
+      - lib/amazonka-core
+      - lib/services/amazonka-sts # amazonka library depends on this
+      - lib/services/amazonka-kinesis
+      - lib/services/amazonka-s3
```

Comment:
Looks like it's time to remove this! 🎉

Reply:
I didn't update this PR, expecting that many of the changes in it would be rejected.
The most up to date version of PR including switch to amazonka-2.0 from hackage is in my fork of this repo (see https://github.com/jhrcek/serverless-haskell/pull/6). I was just waiting for some initial comment from @koterpillar to see how we want to proceed.